### PR TITLE
Patches

### PR DESCRIPTION
**Pull request by friendlyhj**

### CODE REVIEW

File: CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/item/MCItemStack.java [hunk 1]
```diff
@@ -681,8 +681,7 @@ public boolean hasTag() {
     
     @Override
     public void damageItem(int amount, IEntity entity) {
-        if(entity.getInternal() instanceof EntityLivingBase)
-            stack.damageItem(amount, (EntityLivingBase) entity.getInternal());
+        CraftTweakerAPI.logWarning("Actually, the immutable ItemStack is not damageable. Use `mutable` method to make it mutable first!");
```
Comment (jaredlll08) [inline CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/item/MCItemStack.java]:
```suggestion
        CraftTweakerAPI.logWarning("Cannot damage IItemStack, make it mutable with `mutable` first!");
```

**Review by jaredlll08 — CHANGES_REQUESTED** [review]

**Comment by democat3457** [comment]
@friendlyhj attemptDamageItem is still present in the iitemstack expand

**Comment by friendlyhj** [comment]
> @friendlyhj attemptDamageItem is still present in the iitemstack expand

Damn, it is absolutely because I rebased 1.12 branch after the move method commit.

**Comment by democat3457** [comment]
I figured, since the file diff is weird and shows addition of random leftovers from a previous commit

**Pull request merged**